rholang/src/main/scala/coop/rchain/rholang/interpreter/Reduce.scala

```diff
@@ -276,7 +273,7 @@ object Reduce {
         unbundled <- subChan.singleBundle() match {
                       case Some(value) =>
                         if (!value.writeFlag) {
-                          s.raiseError(ReduceError("Trying to send on non-writeable channel."))
+                          s.raiseError[Par](ReduceError("Trying to send on non-writeable channel."))
```

Comment:
I'm failing to see the benefit of removing `import cats._` in favor of specifying the types explicitly here

Reply:
In a subsequent commit, I convert these to direct method calls on the errors. In that commit, type inference fails without the type annotations even if `cats._` is imported.